ui/tests/acceptance/client-detail-test.js

```diff
@@ -75,6 +76,18 @@ test('/clients/:id should list additional detail for the node below the title',
         .includes(node.httpAddr),
       'Address is in additional details'
     );
+    assert.ok(
```

Comment:
Not that you need another testing framework, but `qunit-dom` has been nice to use for tests like this (and you don't need to do all of the trim / includes stuff).

Reply:
My testing backlog includes

  1. qunit-dom
  2. ember-cli-page-object
  3. testing unification
  4. async/await
  5. refactoring how I use mirage now that I know it better

Some day!